core/src/main/kotlin/org/evomaster/core/problem/rest/RestPath.kt

```diff
@@ -183,6 +208,26 @@ class RestPath(path: String) {
         return (0 until this.elements.size).none { other.elements[it] != this.elements[it] }
     }
 
+
+    /**
+     * @return whether this is possible ancestor of [other]
+     *
+     * this is to handle the case eg.
+     * /root/{rootName}/bar/{barName} might be considered as
+     * the ancestor of /root/{rootName}/foo/{fooName}/bar/{barName}
+     *
+     */
+    fun isPossibleAncestorOf(other: RestPath): Boolean {
```

Comment:
why the term `ancestor`? i find it confusing here

Reply:
the doc and the name of the method are modified